great_ape_safe/ape_api/ebtc.py

```diff
@@ -282,6 +344,11 @@ def update_timelock_delay(self, new_delay, use_high_sec=False):
     #### ===== CDP MANAGER ===== ####
 
     def cdpManager_set_staking_reward_split(self, value, use_high_sec=False):
```

Comment:
for this type of txs that are setter that will use `value` will advise to use asserts that are catching early the potential reverts onchain for example in this case:

- setting a value >10k will revert

otherwise may be a "wasted" queue tx imo if nobody catches before hand

Reply:
another though is in the case that some of these values are dynamic based on blockanalytica/treasury/3rd party that modifes them back to the same exact signature due to using the same value, they should be accepting a salt, no?

say, setting from 50 to 60 and then back to 50

Reply:
Good call on the catches, will add.

You are very right on allowing for an optional salt, I missed this. Will add as well and prioritize building a solution for tracking all transactions values.

Reply:
Value catches added: https://github.com/ebtc-protocol/ebtc-multisig/pull/3/commits/157869200187017c7f7fd3be2308e35f1c8c04ac

Reply:
Enabled salts: https://github.com/ebtc-protocol/ebtc-multisig/pull/3/commits/06248d0f4d0d90f313097f580b6ffb94ee2af900